source/exe/main_common.h

```diff
@@ -65,7 +66,8 @@ class MainCommonBase {
                     const AdminRequestFn& handler);
 
 protected:
-  ProcessWide process_wide_; // Process-wide state setup/teardown.
+  ProcessWide process_wide_; // Process-wide state setup/teardown (excluding grpc).
+  Grpc::GoogleGrpcContext google_grpc_context_;
```

Comment:
nit: is it possible for this to be stubbed when there is no Google-gRPC support compiled in? It's a little strange that we still have the context, but the actual calls inside it are compiled out?

Reply:
OK fair enough; can make that change. Do you want to consider merging this in first though and I will commit to cleaning it up in a follow-up, so we can de-flake CI?

Reply:
Sure sounds good.